src/SAWScript/Yosys/Utils.hs

```diff
@@ -19,6 +19,7 @@ import Control.Monad (forM, foldM)
 import Control.Monad.IO.Class (MonadIO(..))
 import Control.Exception (Exception, throw)
 import Control.Monad.Catch (MonadThrow)
+import Control.Monad.Except
```

Comment:
I think this import is unnecessary. Removing it ought to make the CI stop complaining.